apparmor.d/abstractions/systemd-common

```diff
@@ -14,6 +20,7 @@
         @{PROC}/cmdline r,
         @{PROC}/sys/kernel/osrelease r,
         @{PROC}/sys/kernel/random/boot_id r,
+  owner @{PROC}/filesystems r,
```

Comment:
Same what program need this?

Reply:
Hmm, this should already be part of abstraction/base...